### ISSUE TITLE
Location of receipt.json after deployment is slightly confusing 

### ISSUE THREAD

**Issue by dcroote** [api3dao/airnode#1515]
Running the deployer image
```sh
docker run -it --rm -v /home/derek/api3/airnode/packages/airnode-examples/integrations/coingecko-testable:/app/config api3/airnode-deployer-dev:3a69963ff07f769e1903707b070997c0e1d4a6cc deploy
```
Results in a successful deployment with the following (trimmed) output:
```sh
ℹ Outputted config/receipt.json
  This file does not contain any sensitive information.
```

What is slightly confusing is that there isn't a local `config/receipt.json` - the `receipt.json` file is written to the **image's** `/app/config` directory, which is mounted to my local directory (and hence `receipt.json` is in my mounted directory).

@amarthadan  WDYT?

**Comment by amarthadan** [comment]
Right. Unfortunately, there's no way for the application inside the container to know that the directory is mounted somewhere. It doesn't even know it's running in the container really.
We can omit the path in the output entirely but I don't think that will help much.

**Comment by dcroote** [comment]
Good points. it may be slightly clearer to just say `Saved receipt.json` i.e. replace the path with just the filename?

**Comment by amarthadan** [comment]
I think it's quite common for containerized apps to log paths that are only valid within the container itself. The user using Docker should expect that. What do others think? @api3dao/airnode

**Comment by patriksimurka** [comment]
Agree, but that being said, we could maybe do something like
```
Outputted config/receipt.json to the mounted directory
```

**Comment by amarthadan** [comment]
> Agree, but that being said, we could maybe do something like
> 
> ```
> Outputted config/receipt.json to the mounted directory
> ```

But that's not technically correct because you can run the deployer outside the container and the message wouldn't make sense.

**Comment by dcroote** [comment]
What if we logged the absolute path instead? This would work both in and outside the container and it would remove the user needing to inferring the container's workdir was `/app` which, when combined with the relative `config/receipt.json`, yields the file in the mounted directory.
